litmus/skel.ml

```diff
@@ -413,6 +413,8 @@ module Make
           if have_timebase then O.f "#define DELTA_TB %s" delta
         end ;
         O.o "/* Includes */" ;
+        let module I = ObjUtil.Insert(Cfg) in
+        I.insert_when_exists O.o "intrinsics.h" ;
```

Comment:
The functor `Objutil.Insert` has already been applied above and the result is in scope as `Insert`:
```suggestion
        Insert.insert_when_exists O.o "intrinsics.h" ;
```
 Moreover the same line  should be added in the file `litmus/preSi.ml`  (line 416) for the command-line option `-mode presi` (and `-mode vmsa`) to work.